CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
+### ✅ Added
+- Role value in the user display info
```

Comment:
Does it make sense to add something to the DemoApp?

Reply:
Customers would use it to show role badge in the avatar, but we don't have such design for the demo app. Not sure if we can use it elsewhere? Open to ideas.

Reply:
Was thinking about adding something in the channel's info besides users' names, but now not sure it's worth it, to be honest. Let's not overcomplicate this 👍